source/includes/documentation/_vue_js_elements.md

```diff
@@ -5,25 +5,81 @@
 [Vuetify documentation page](https://vuetifyjs.com/en/components/alerts/)
 
 ```java
-  @Test
-  public void alertsWithPropsHaveProperCssProps(){
-      redAlert.has().text("I'm an alert with a top border and red color");
-      redAlert.has().css("background-color", Colors.RED_LIGHTEN_2.value());
-      redAlert.has().cssClass("v-alert--border-top");
-  }
+@Test(dataProvider = "textAlertsTestsData",
+  dataProviderClass = AlertsTestsDataProvider.class)
+public void textAlertsTest(int index, String alertText) {
+	basicAlerts.get(index).show();
+	basicAlerts.get(index).has().text(alertText);
+}
+
+@Test(dataProvider = "colorAlertsTestsData",
+  dataProviderClass = AlertsTestsDataProvider.class)
+public void colorAlertsTest(int index, String color, String backgroundColor) {
+  basicAlerts.get(index).show();
+  basicAlerts.get(index).has().color(color)
+    .and().backgroundColor(backgroundColor);
+}
 ```
 
 - __Java__: _com.epam.jdi.light.vuetify.elements.common.Alert.java_
 
 ![Alerts example](../../images/vuetify/alerts.png)
 
-|Method | Description | Return Type
---- | --- | ---
-**is()** | Returns Assert class | TextAssert
-**getValue()** | Returns element value | String
-
-For examples of usage see: [Custom vuetify alert example](https://github.com/jdi-testing/jdi-light/blob/3118-implement-alerts/jdi-light-vuetify-tests/src/main/java/io/github/com/custom/CustomAlert.java)
-and [JDI vuetify page tests for alerts](https://github.com/jdi-testing/jdi-light/blob/3118-implement-alerts/jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/AlertsTests.java).
+__Vuetify v2.6.14__ code example:
+```html
+<template>
+  <div>
+    <v-alert
+      dense
+      type="info"
+    >
+      I'm a dense alert with a <strong>type</strong> of info
+    </v-alert>
+    <v-alert
+      dense
+      text
+      type="success"
+    >
+      I'm a dense alert with the <strong>text</strong> prop and a <strong>type</strong> of success
+    </v-alert>
+  </div>
+</template>
```

Comment:
Please, change according to Vladimir's comments:
in a documentation near highlighted comment please also emphasize a Selenium locator findby 
![image](https://user-images.githubusercontent.com/55189259/212975403-347b94e2-445c-4b2b-96ba-2b5e4f8339fa.png)